### DISCUSSION TITLE
IP Change

### DISCUSSION THREAD

**Discussion by wegi82** [pail23/stiebel_eltron_isg_component#200] [Q&A] [answered]
Is there an easy way to change the IP adress? After the reboot of my system the Stiebel Eltron ISG has a new IP and i can not find the possibility to change the IP in Home Assistant.

**Comment by Padanian** [comment]
I suggest to set the ISG a static IP address, and disable DHCP

**Reply by wegi82** [reply]
I know that possibility but i want to know the other way round.

**Comment by xylone** [comment, marked as answer]
You can change the IP address in the config file core.config_entries in the folder <config dir>/.storage

**Reply by wegi82** [reply]
Thanks a lot! Works fine!

**Reply by Kunibert-007** [reply]
Have the same doing (change IP-Adress) but can't find the entry to change. Can you give me a hint where to find the file and the entry to change?
Thank you  @xylone xylone or @wegi82 
<img width="212" alt="image" src="https://github.com/user-attachments/assets/7ca2997e-0707-412b-bfc8-2b387e892615">

**Reply by xylone** [reply]
You are looking in the plugin directory. That’s not the right one. You have to go to the root first.

**Reply by Kunibert-007** [reply]
Well, I don't know where to search. That's my root folder:
<img width="197" alt="image" src="https://github.com/user-attachments/assets/fa7aad2d-49ad-4176-8187-655465ea7c4b">
There is no /.storage-Folder or something like this.

**Reply by xylone** [reply]
Files and directories starting with a dot, are hidden. So you need to make sure you see hidden files or use a terminal with vi or nano

**Comment by mbeckerkassel** [comment]
in my case configuring it with servicewelt.local works
